metacat-connector-hive/src/main/java/com/netflix/metacat/hive/connector/ConnectorTableServiceImpl.java

```diff
@@ -0,0 +1,140 @@
+/*
+ *  Copyright 2016 Netflix, Inc.
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ */
+
+package com.netflix.metacat.hive.connector;
+
+import com.google.inject.Inject;
+import com.netflix.metacat.common.MetacatRequestContext;
+import com.netflix.metacat.common.QualifiedName;
+import com.netflix.metacat.common.dto.Pageable;
+import com.netflix.metacat.common.dto.Sort;
+import com.netflix.metacat.common.dto.TableDto;
+import com.netflix.metacat.common.server.connectors.ConnectorTableService;
+import com.netflix.metacat.common.server.exception.TableAlreadyExistsException;
+import com.netflix.metacat.common.server.exception.TableNotFoundException;
+import com.netflix.metacat.hive.connector.converters.HiveMetacatConverters;
+import org.apache.hadoop.hive.metastore.api.Table;
+import org.apache.thrift.TException;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.List;
+
+/**
+ * Hive base connector base service impl.
+ *
+ * @author zhenl
+ */
+public class ConnectorTableServiceImpl implements ConnectorTableService {
```

Comment:
This class should be named `HiveConnectorTableService` or `HiveConnectorTableServiceImpl` cause this name is too generic as there will be many different implementations of the `ConnectorTableService` within our different connectors. In the JDBC one I have it named `JdbcConnectorTableService`